authentication/auth_method_azure_cli_token.go

```diff
@@ -179,6 +182,23 @@ func obtainAuthorizationToken(endpoint string, subscriptionId string) (*cli.Toke
 	return &token, nil
 }
 
+// obtainSubscription return a subscription object of the specified subscriptionId.
+// If the subscriptionId is empty, it returns the default subscription.
+func obtainSubscription(subscriptionId string) (*cli.Subscription, error) {
+	var sub cli.Subscription
+	cmd := make([]string, 2, 4)
```

Comment:
this starts at 3 now, and we can remove the max (since it's invalid)

```suggestion
	cmd := make([]string, 0)
```